src/sqlancer/questdb/QuestDBSchema.java

```diff
@@ -142,6 +147,8 @@ public String toString() {
                 }
             case BOOLEAN:
                 return Randomly.fromOptions("BOOLEAN");
+            case SYMBOL:
+                return Randomly.fromOptions("SYMBOL");
```

Comment:
Here (and above), you can directly return the string (i.e., `"SYMBOL"`). `Randomly.fromOptions` returns an element from the variadic arguments given. Since only one argument is given, the method directly returns the object.